# Module src_wingbase translation file for language=en
# Copyright (C) 2016 Wingware
# Wingware Python IDE -- www.wingware.com
#
msgid ""
msgstr ""
"Project-Id-Version: 6.0.0\n"
"POT-Creation-Date: 2025-01-13 08:22-0500\n"
"PO-Revision-Date: 2016-06-08 17:19 EDT\n"
"Last-Translator: Wingware <support@wingware.com>\n"
"Language-Team: None\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: text/plain\n"
"Generated-By: pygettext.py 1.5\n"

#: src/wingbase/containers.py:276 src/wingbase/remoteclient.py:403
msgid "File exceeds configured maximum size (%.1fMB>%.0fMB)"
msgstr ""

#: src/wingbase/coverageutils.py:338
msgid "Output received was:"
msgstr ""

#: src/wingbase/coverageutils.py:340
msgid "No output was received from coverage.py."
msgstr ""

#: src/wingbase/coverageutils.py:350
msgid "No report was generated by coverage.py."
msgstr ""

#: src/wingbase/dlog.py:293
msgid "Wing Debug Log File\n"
msgstr ""

#: src/wingbase/dlog.py:294
msgid "This file can be used to diagnose problems in debugger internals\n"
msgstr ""

#: src/wingbase/dlog.py:295
msgid "For help, contact support@wingware.com\n"
msgstr ""

#: src/wingbase/dlog.py:300
msgid "Warning:  Log file does not exist:\n"
msgstr ""

#: src/wingbase/dlog.py:302
msgid "Warning:  and could not be created; logging disabled\n"
msgstr ""

#: src/wingbase/dlog.py:305
msgid "Warning:  Log file is not a file:\n"
msgstr ""

#: src/wingbase/dlog.py:307
msgid "Warning:  Logging disabled\n"
msgstr ""

#: src/wingbase/filefilter.py:60
msgid "Wild Card on File Name"
msgstr ""

#: src/wingbase/filefilter.py:61
msgid "Mime Type"
msgstr ""

#: src/wingbase/filefilter.py:62
msgid "Wild Card on Directory Name"
msgstr ""

#: src/wingbase/filefilter.py:428 src/wingbase/filefilter.py:433
#: src/wingbase/filefilter.py:440
msgid "Invalid or empty filter %s spec pos=%i: '%s'"
msgstr ""

#: src/wingbase/filefilter.py:435
msgid "Bad mime type in filter %s spec pos=%i: '%s'"
msgstr ""

#: src/wingbase/filefilter.py:444
msgid "Unknown filter %s spec type pos=%i: '%s'"
msgstr ""

#: src/wingbase/mime.py:136
msgid "Wing Project"
msgstr ""

#: src/wingbase/mime.py:137
msgid "Python Interface"
msgstr ""

#: src/wingbase/mime.py:540 src/wingbase/mime.py:542 src/wingbase/mime.py:617
msgid "%s File"
msgstr ""

#: src/wingbase/remoteclient.py:1033
msgid "Missing remote host configuration for %s"
msgstr ""

#: src/wingbase/remoteclient.py:1298
msgid ""
"Remote server %s failed to connect (hostname=%s, winghome=%s, pyexec=%s)"
msgstr ""

#: src/wingbase/remoteclient.py:1308
msgid "Remote server to %s is unused -- closing connection"
msgstr ""

#: src/wingbase/remoteclient.py:1741
msgid ""
"Reverse SSH tunnel startup timed out -- connection ID not seen after %.2f "
"seconds"
msgstr ""

#: src/wingbase/remoteclient.py:1936
msgid "Reverse tunnel to %s (%i<->%i) failed after %i retries"
msgstr ""

#: src/wingbase/remoteclient.py:1941
msgid "Reverse tunnel to %s (%i<->%i) is unused -- closing connection"
msgstr ""

#: src/wingbase/textutils.py:1123
msgid "Unable to convert to unicode.  Tried encodings: %s."
msgstr ""

#: src/wingbase/updateutils.py:173
msgid "Corrupt catalog: Missing | in line: "
msgstr ""

#: src/wingbase/updateutils.py:185
msgid "Corrupt catalog: Invalid line type: "
msgstr ""

#: src/wingbase/updateutils.py:334
msgid "Version spec on line %i is too short"
msgstr ""

#: src/wingbase/updateutils.py:338
msgid "Bad version %s on line %i"
msgstr ""

#: src/wingbase/updateutils.py:350
msgid "Bad release type %s on line %i"
msgstr ""

#: src/wingbase/updateutils.py:354
msgid "Bad release flag %s on line %i"
msgstr ""

#: src/wingbase/updateutils.py:359
msgid "Line %i lists summary before version"
msgstr ""

#: src/wingbase/updateutils.py:362
msgid "Bad change summary on line %i"
msgstr ""

#: src/wingbase/updateutils.py:365
msgid "Bad product type on line %i"
msgstr ""

#: src/wingbase/updateutils.py:368
msgid "Bad change type on line %i"
msgstr ""

#: src/wingbase/updateutils.py:502
msgid "Some unpatchable files changed:"
msgstr ""

#: src/wingbase/updateutils.py:506
msgid "No changes for the update"
msgstr ""

#: src/wingbase/updateutils.py:539
msgid "Could not create update tarfile:"
msgstr ""

#: src/wingbase/updateutils.py:645
msgid "Updates are corrupt and need to be removed and reinstalled"
msgstr ""

#: src/wingbase/updateutils.py:695
msgid "Failed to delete old busy update: %s"
msgstr ""

#: src/wingbase/updateutils.py:698
msgid "Possibly concurrent update/merge on %s"
msgstr ""

#: src/wingbase/updateutils.py:706
msgid "Renaming %s failed; skipping update merge"
msgstr ""

#: src/wingbase/updateutils.py:739
msgid "Failed to merge update directories: %s -> %s; aborting merge"
msgstr ""

#: src/wingbase/updateutils.py:917
msgid "Missing file: %s"
msgstr ""

#: src/wingbase/updateutils.py:921
msgid "Invalid Type: %s (expected file but found non-file)"
msgstr ""

#: src/wingbase/updateutils.py:923
msgid "Invalid Type: %s (expected directory but found non-directory)"
msgstr ""

#: src/wingbase/updateutils.py:928
msgid "Read failed:  %s"
msgstr ""

#: src/wingbase/updateutils.py:931
msgid "Changed file: %s"
msgstr ""
